Makefile

```
@@ -1,10 +1,11 @@
+LUA_VER ?= 5.2
 SONAME   = lua_sysctl
 BUILDDIR = build
 SOLIB    = ${BUILDDIR}/sysctl.so
 DESTDIR ?= sysctl
 
 LDFLAGS += -shared -Wl,-soname,${SONAME}
-CFLAGS  += -Wall -Wextra -fPIC `pkg-config --cflags lua-5.2`
+CFLAGS  += -Wall -Wextra -fPIC `pkg-config --cflags ${LUA_VER}`
```

This is not working as-is for me:
```console
% make
install -m 755 -d build
cc -O2 -pipe -Wall -Wextra -fPIC `pkg-config --cflags 5.2` -o build/sysctl.so  -shared -Wl,-soname,lua_sysctl src/lua_sysctl.c
Package 5.2 was not found in the pkg-config search path.
Perhaps you should add the directory containing `5.2.pc'
to the PKG_CONFIG_PATH environment variable
Package '5.2', required by 'virtual:world', not found
src/lua_sysctl.c:60:10: fatal error: 'lua.h' file not found
#include "lua.h"
         ^~~~~~~
1 error generated.
*** Error code 1

Stop.
make: stopped in /usr/home/alex/github/lua-sysctl
```

```suggestion
CFLAGS  += -Wall -Wextra -fPIC `pkg-config --cflags lua-${LUA_VER}`
```